src/ResourceManager/Compute/Commands.Compute/Generated/VirtualMachineScaleSetRollingUpgrade/VirtualMachineScaleSetRollingUpgradeCancelMethod.cs

```diff
@@ -79,8 +79,7 @@ protected void ExecuteVirtualMachineScaleSetRollingUpgradeCancelMethod(object[]
             string resourceGroupName = (string)ParseParameter(invokeMethodInputParameters[0]);
             string vmScaleSetName = (string)ParseParameter(invokeMethodInputParameters[1]);
 
-            var result = VirtualMachineScaleSetRollingUpgradesClient.Cancel(resourceGroupName, vmScaleSetName);
-            WriteObject(result);
+            VirtualMachineScaleSetRollingUpgradesClient.Cancel(resourceGroupName, vmScaleSetName);
```

Comment:
breaking change?

Reply:
Same thing -= it looks like the work for the cmdlet is done in the method below, so what is the purpose of this change?